app/renderer/js/utils/enterprise-util.js

```diff
@@ -25,17 +26,17 @@ class EnterpriseUtil {
 	reloadDB() {
 		let enterpriseJsonPath = null;
 		if (process.platform === 'win32') {
-			enterpriseJsonPath = 'C:\\Program Files\\Zulip\\enterprise_config.json';
+			enterpriseJsonPath = 'C:\\Program Files\\Zulip-Desktop-Config\\enterprise_config.json';
 		} else {
-			enterpriseJsonPath = '/etc/zulip/enterprise_config.json';
+			enterpriseJsonPath = '/etc/zulip-desktop-config/enterprise_config.json';
```

Comment:
These changes should have been part of the previous change.

@kanishk98 [these docs](https://zulip.readthedocs.io/en/latest/git/fixing-commits.html#fixing-older-commits) should help you out with this stuff if you are having trouble or just not confident in doing git rebase.

Reply:
Sure sure. I forgot to rebase these changes.